sdk/samples/iot/docs/how_to_iot_hub_esp32.md

```diff
@@ -0,0 +1,311 @@
+        RODATA : 2152  ) / 81920 - constants             (global, static) in RAM/HEAP
+        BSS    : 26528 )         - zeroed variables      (global, static) in RAM/HEAP
+        Sketch uses 392272 bytes (37%) of program storage space. Maximum is 1044464 bytes.
+        Global variables use 30040 bytes (36%) of dynamic memory, leaving 51880 bytes for local variables. Maximum is 81920 bytes.
+        /home/user/.arduino15/packages/esp8266/tools/python3/3.7.2-post1/python3 /home/user/.arduino15/packages/esp8266/hardware/esp8266/2.7.1/tools/upload.py --chip esp8266 --port /dev/ttyUSB0 --baud 230400 --before default_reset --after hard_reset write_flash 0x0 /tmp/arduino_build_826987/azure_iot_hub_telemetry.ino.bin
+        esptool.py v2.8
+        Serial port /dev/ttyUSB0
+        Connecting....
+        Chip is ESP8266EX
+        Features: WiFi
+        Crystal is 26MHz
+        MAC: dc:4f:22:5e:a7:09
+        Uploading stub...
+        Running stub...
+        Stub running...
+        Changing baud rate to 230400
+        Changed.
+        Configuring flash size...
+        Auto-detected Flash size: 4MB
+        Compressed 396432 bytes to 292339...
+
+        Writing at 0x00000000... (5 %)
+        Writing at 0x00004000... (11 %)
+        Writing at 0x00008000... (16 %)
+        Writing at 0x0000c000... (22 %)
+        Writing at 0x00010000... (27 %)
+        Writing at 0x00014000... (33 %)
+        Writing at 0x00018000... (38 %)
+        Writing at 0x0001c000... (44 %)
+        Writing at 0x00020000... (50 %)
+        Writing at 0x00024000... (55 %)
+        Writing at 0x00028000... (61 %)
+        Writing at 0x0002c000... (66 %)
+        Writing at 0x00030000... (72 %)
+        Writing at 0x00034000... (77 %)
+        Writing at 0x00038000... (83 %)
+        Writing at 0x0003c000... (88 %)
+        Writing at 0x00040000... (94 %)
+        Writing at 0x00044000... (100 %)
+        Wrote 396432 bytes (292339 compressed) at 0x00000000 in 13.0 seconds (effective 243.4 kbit/s)...
+        Hash of data verified.
+
+        Leaving...
+        Hard resetting via RTS pin...
+        ```
+
+        </p>
+        </details>
+
+8. Monitor the MCU (microcontroller) locally via the Serial Port.
+
+    - Go to menu `Tools`, `Serial Monitor`.
+
+        If you perform this step right away after uploading the sketch, the serial monitor will show an output similar to the following upon success:
+
+        ```text
+        Connecting to WIFI SSID buckaroo
+        .......................WiFi connected, IP address:
+        192.168.1.123
+        Setting time using SNTP..............................done!
+        Current time: Thu May 28 02:55:05 2020
+        Client ID: mydeviceid
+        Username: myiothub.azure-devices.net/mydeviceid/?api-version=2018-06-30&DeviceClientType=c%2F1.0.0
+        Password: SharedAccessSignature sr=myiothub.azure-devices.net%2Fdevices%2Fmydeviceid&sig=0VFwGiXlIVPeCmPStJ4Fb1wbS8o2W8p1vzIOt%2B8K2eE%3D&se=1590620105
```

Comment:
i feel obligated to double check this isn't real

Reply:
ehh which i guess doesn't matter since it's expired by this point

Reply:
Yeah, they are not real. Thanks for checking though.